### DISCUSSION TITLE
Is there a way to tell the version number from the binary?

### DISCUSSION THREAD

**Discussion by MrTrebron** [sqlpage/SQLPage#85] [General]
Hi

Is there a way to tell the version number from the binary?
For the Windows exe I don't find a way.

Thanks
Norbert

**Comment by lovasoa** [comment]
I know it's really not obvious, but the version number is rendered on hover on the footer message in the default shell template.

![image](https://github.com/lovasoa/SQLpage/assets/552629/d667a961-72b8-4526-80f2-c0b8a5b9e427)
